app/controllers/discover.js

```diff
@@ -121,6 +127,39 @@ export default ApplicationController.extend({
         });
     },
 
+    transformTypes(obj) {
+        if (typeof (obj) !== 'object') {
+            return obj;
+        }
+        var keys = Object.keys(obj);
+        var n = keys.length;
+
+        var lowKey;
+        while (n--) {
+            var key = keys[n];
+            if (key === (lowKey = key.replace(/([A-Z])/g, ' $1').trim().toLowerCase())) {
```

Comment:
You only need the `toLowerCase` here. The `replace` isn't actually doing anything

```javascript
'Foo Bar'.replace(/([A-Z])/g, ' $1').trim()
> "Foo  Bar"
```